### PR TITLE
update to beta1

### PR DESCRIPTION
**Pull request by gimler**
- update to beta1

I am not shure for what geoip would be used?

Also i would prefer to remove all the `PROFILE` stuff and refer in the docs to the full example. And how you can use a volume to use your own config.

Because i think mostly you would use your own config.

### CODE REVIEW

**Comment by athieriot** [comment]
Hi @gimler,

  Thank you for the Pull Request.

   I agree that the ability to provide your own config file is what will probably be the main way to configure the container.
   I started this project as a copy-paste of docker-topbeat and most of it doesn't make sense now.

   I'll probably still provide a way to configure it via environment variable in the future but that's another topic.

   `alpine-glibc` is very sweet as well :)
   And you would be fine to remove the GeoIP files as I suspect it's not used by `metricbeat` anymore.

   However, would you mind updating the Readme to reflect the change of usage (including any volume statement needed)?

Thanks !

File: Dockerfile [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
 
-ENV METRICBEAT_VERSION=5.0.0-alpha4
+ENV METRICBEAT_VERSION=5.0.0-beta1
 
-RUN apk update && \
-    apk add \
+RUN apk add --no-cache \
       ca-certificates \
-      curl && \
-    rm -rf /var/cache/apk/*
```
Comment (athieriot) [inline Dockerfile:10]: Don't we still need that statement?

Comment (gimler) [reply Dockerfile:10]: no because we use `--no-cache` see https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache

Comment (athieriot) [reply Dockerfile:10]: Oh nice. Perfect

**Comment by gimler** [comment]
ready for merge ;)

**Comment by athieriot** [comment]
That looks perfect.
I am giving it a little spin right now just to make sure I still understand how it works :)

Should be released today.

Thank you very much for your time

**Comment by athieriot** [comment]
Hi !

5.0.0-beta1 is available in the registry. https://hub.docker.com/r/athieriot/metricbeat/tags/

In the future, if I got around to do it (Time I mean !), I still like to provide a way to configure the container via Env Variable (Had this idea of using this: https://github.com/tests-always-included/mo as Metricbeat configuration won't play well with my old split files).

I will also give a try at applying those advice and update the Readme accordingly: https://www.elastic.co/guide/en/beats/metricbeat/5.x/_running_metricbeat_in_a_container.html

**Comment by gimler** [comment]
thx.

:+1:  readme is interresting i can also take a look into it.

:-1: env. i think it would be better when the user read the conf file because it is well documented. Env would be a copy and waste of time i think. Also it is all in one file.

**Pull request merged**